ncm-grub/src/main/perl/grub.pm

```diff
@@ -74,6 +77,66 @@ sub grubbyArgsOptions {
 
 }
 
+sub Configure_password {
```

Comment:
add comment what this method does. there's also no need for a `Configure_` prefix